Makefile

```diff
@@ -266,6 +265,10 @@ clean: ## Remove all generated files
 clean-bazel: ## Remove all generated bazel symlinks
 	bazel clean
 
+.PHONY: clean-bazel-mocks
+clean-bazel-mocks: ## Remove all generated bazel mocks files
+	find bazel-bin/pkg -name '*_mock.go' -type f -delete
```

Comment:
Can we use `bazel clean` here instead of `find -delete`?

Reply:
The `make clean-bazel` target is too aggressive for this particular issue, this removes only the mocks in cache.

Reply:
Sorry for not being clear. I mean can we use `bazel clean <some-target>`? not `bazel clean` everything.

Reply:
I don't think clean accepts any arguments from what I can see

Reply:
OK. I wasn't able to find examples that we can use here.
I am 👍 w/ this change.